http.slide

```diff
@@ -28,3 +28,30 @@ pawel.kowalak@gmail.com
     "temp": 18,
     "windy": false
   }
+
+* Simplest HTTP server
+
+.code http/w.go
+
+* ...with error handling
```

Comment:
Yeah, I just didn't know how to include few blocks at once. Probably only with OMIT flags. I'll clean it up.